Remotion/Web/ClientScript/Scripts/DatePicker.ts

```diff
@@ -101,7 +110,7 @@ class DatePicker
     current.DatePicker.remove();
     if (current.TargetDocument != null)
     {
-      $(current.TargetDocument).unbind('click', current.ClickHandler!); // TODO RM-7647: DatePick.DatePicker_CloseDatePicker removes all click handlers from target element if no click handler was specified for the DatePicker
+      current.TargetDocument.removeEventListener('click', current.ClickHandler!);
```

Comment:
I forgot that we were removing jQuery and since he issue was a wrong invocation of a jQuery method it is not "fixed" because we no longer use jQuery. I think obsolete fits better than fixed.

Reply:
Hmm... The bug existed before this change and no longer exists after this change but the functionality that was buggy still exists. This would make the bug fixed. 

Obsolete is used when the entire feature has been removed. This means that the bug wasn't actually fixed but rather the buggy feature is no longer available.

Maybe a different way to look at things would be to ask if the bugfix can be tested/QA'd. If the behavior still exists but now works without exhibiting the bug, it's fixed. If the correct behavior can no longer be tested because the feature's gone, the bug's obsolete.

Reply:
Oh, and of course, a link to the issue that fixes the bug is needed, too. Updated Jira already.